deploy_preservica_config.tf

```diff
@@ -31,6 +32,23 @@ module "preservica_config_sns" {
   }
 }
 
+module "dr2_preservica_config_sns" {
+  source = "git::https://github.com/nationalarchives/da-terraform-modules//sns"
+  sns_policy = templatefile("${path.module}/templates/sns/s3_notifications_policy.json.tpl", {
+    topic_name  = "${local.environment}-preservica-config"
+    bucket_name = "${local.environment}-dr2-preservica-config"
+    account_id  = data.aws_caller_identity.current.account_id
+  })
+  tags = {
+    Name = "Preservica Config SNS"
+  }
+  topic_name  = "${local.environment}-preservica-config"
```

Comment:
Since there are 2 instances of this, should we save it to a local variable?

Reply:
Yeah, we can do. I've added in a variable to `preservica_config_sns_name` and `preservica_config_sns_name_old` I know they're the same as the other names but it's nice to keep things seperate.